py3status/modules/xrandr.py

```diff
@@ -28,7 +28,9 @@
         when the module starts (saves you from having to configure xorg)
         (default None)
     format: display format for xrandr
-        (default '{output}')
+        (default '[{output}]')
```

Comment:
why has the format changed?  I don't see it making a difference can't we just keep the old and do 

```
full_text = self.py3.safe_format(self.format, {'output': ''})
```